samples/unit-tests/chart/panning/demo.js

```diff
@@ -356,66 +356,49 @@ QUnit.test('Stock panning (#6276, #21319)', function (assert) {
+    'Ordinal axis panning.',
     function (assert) {
-        var chart = Highcharts.stockChart('container', {
-            xAxis: {
-                min: Date.UTC(2020, 1, 6),
-                max: Date.UTC(2020, 1, 9)
+        const data = [];
+
+        for (let i = 0; i < 100000; i++) {
+            data.push([400 + i, 1]);
+        }
+        const chart = Highcharts.stockChart('container', {
+            chart: {
+                events: {
+                    load: function () {
+                        this.xAxis[0].setExtremes(2000, 45000);
+                    }
+                }
             },
-            series: [
-                {
-                    data: [
-                        {
-                            x: Date.UTC(2020, 1, 1),
-                            y: 10
-                        },
-                        {
-                            x: Date.UTC(2020, 1, 2),
-                            y: 11
-                        },
-                        {
-                            x: Date.UTC(2020, 1, 3),
-                            y: 12
-                        },
-                        {
-                            x: Date.UTC(2020, 1, 4),
-                            y: 14
-                        },
-                        {
-                            x: Date.UTC(2020, 1, 5),
-                            y: 15
-                        },
-                        {
-                            x: Date.UTC(2020, 1, 6),
-                            y: 16
-                        },
-                        {
-                            x: Date.UTC(2020, 1, 7),
-                            y: 14
-                        },
-                        {
-                            x: Date.UTC(2020, 1, 8),
-                            y: 15
-                        },
-                        {
-                            x: Date.UTC(2020, 1, 9),
-                            y: 16
-                        }
-                    ]
+            series: [{
+                type: 'column',
+                data: data,
+                dataGrouping: {
+                    units: [[
+                        'second',
+                        [1]
+                    ]]
                 }
-            ]
+            }]
         });
 
-        var controller = new TestController(chart),
+        const controller = new TestController(chart),
             initialMin = chart.xAxis[0].min;
 
         controller.pan([100, 200], [200, 200]);
 
         assert.notEqual(
             initialMin,
             chart.xAxis[0].min,
-            'Chart should pan horizontally.'
+            'Chart should pan horizontally, when data is equally spaced #13334.'
+        );
+
+        assert.equal(
+            chart.xAxis[0].min,
+            0,
+            `It should be possible to pan to the axis minimum in a data grouped
+            ordinal column chart, #21524.`
```

Comment:
When testing grouped data, please set `forced=true`. And keep the old test. What I meant in the previous review is: to move new test to the old test, so we will reuse the chart (it takes most of the time in our tests). Something like this:

```ts
const chart = new chart(..);
controller.pan(..);
assert.notEqual(..);

chart.series[0].update({ data, dataGrouping });
controller.pan(..);
assert.equal(..);
```